spec/jekyll-feed_spec.rb

```diff
@@ -116,7 +124,11 @@
     end
 
     it "doesn't include the item's excerpt if blank" do
-      post = feed.items.first
+      if Gem::Version.new(Jekyll::VERSION) > Gem::Version.new('3')
```

Comment:
Before Jekyll 3.0, it looks like draft were generated with today date path, therefore leading first item was the draft instead of the real first post.